src/python.c

```diff
@@ -318,11 +325,17 @@ void cpy_log_exception(const char *context) {
     typename = "NamelessException";
   if (message == NULL)
     message = "N/A";
-  Py_BEGIN_ALLOW_THREADS ERROR("Unhandled python exception in %s: %s: %s",
```

Comment:
This unfortunate formatting is very likely clang-format's fault. Can we add a semi-colon after the macro to put a stop to this?

Reply:
I'll add these semi-colons...